Tasks/app-store-release/app-store-release.ts

```diff
@@ -258,6 +259,7 @@ async function run() {
             deliverCommand.argIf(teamName, ['-e', teamName]);
             deliverCommand.argIf(shouldSubmitForReview, ['--submit_for_review', 'true']);
             deliverCommand.argIf(shouldAutoRelease, ['--automatic_release', 'true']);
+            deliverCommand.argIf(usesIdfa, ['--submission_information', `'{'add_id_info_uses_idfa': ${usesIdfa}}'`]);
```

Comment:
This change could be unobvious - fastlane command tools doesn't support repeated flags, so if fastlaneArguments is defined - '--submission_information' there could override this value